pkg/helmcertifier/checks/checks.go

```diff
@@ -57,7 +59,21 @@ func HasReadme(uri string) (Result, error) {
 }
 
 func ContainsTest(uri string) (Result, error) {
-	return notImplemented()
+	c, err := loadChartFromURI(uri)
+	if err != nil {
+		return Result{}, err
+	}
+
+	containTest := false
+	for _, f := range c.Templates {
+		if strings.HasPrefix(f.Name, "templates/tests/") {
```

Comment:
It should validate that at least one YAML test file is present in the directory, so it should include the suffix as well.

Reply:
done